packages/dart_frog_cli/lib/src/dev_server_runner/dev_server_runner.dart

```diff
@@ -142,7 +147,12 @@ class DevServerRunner {
 
   Future<void> _codegen() async {
     logger.detail('[codegen] running pre-gen...');
-    var vars = <String, dynamic>{'port': port};
+    final address = this.address;
+    logger.detail('Starting devsercer on host ${address?.address}');
```

Comment:
It seems that "devsercer" is a typo.

Here are two possible fixes:

```suggestion
    logger.detail('Starting development server on host ${address?.address}');
```
or

```suggestion
    logger.detail('Starting devserver host ${address?.address}');
```

I prefer the former since by avoiding to use abbreviations we guarantee a larger number of users will not find it ambiguos, but I don't feel strong about it here.